com.unity.render-pipelines.universal/Editor/ShaderGraph/Targets/UniversalUnlitSubTarget.cs

```diff
@@ -54,6 +54,12 @@ public override void ProcessPreviewMaterial(Material material)
                 material.SetFloat(Property.ZTest, (float)target.zTestMode);
             }
 
+            // We always need these properties regardless of whether the material is allowed to override
+            // Queue control & offset enable correct automatic render queue behavior
+            // Control == 0 is automatic, 1 is user-specified render queue
+            material.SetFloat(Property.QueueOffset, 0.0f);
```

Comment:
Can you make these queue control constants actual constants or an enum or something? Would help a lot with readability

Reply:
Done (latest commit)